### PR TITLE
feat: Make android-studio optional

### LINKED ISSUE #1293: Android setup forces android studio install
**Describe the bug**
When setting up for flutter or similar android studio is not needed. It is forced by the android integration though.

I dont know if this is by design. But it feels overlooked :) Thank you for the integration - that convenience is MAJOR.

**Version**
 devenv 1.0.7 (x86_64-linux) - no flake

### PR DESCRIPTION
**Pull request by k3yss**
Fixes #1293

### CODE REVIEW

**Comment by domenkozar** [comment]
Should it be then off in the example?

**Comment by k3yss** [comment]
Disabled it! @domenkozar

File: src/modules/integrations/android.nix [hunk 1]
```diff
@@ -212,7 +212,16 @@ in
...
 
-    android-studio = lib.mkOption {
+    android-studio.enable = lib.mkOption {
+      type = lib.types.bool;
+      default = true;
+      defaultText = "true";
+      description = ''
+        Weather to enable the installation of Android Studio package.
+      '';
+    };
```
Comment (sandydoo) [inline src/modules/integrations/android.nix:222]:
```suggestion
    android-studio.enable = lib.mkEnableOption "the installation of Android Studio";
```

**Review by sandydoo — CHANGES_REQUESTED** [review]

**Comment by domenkozar** [comment]
There are conflicts to be resolved

**Comment by k3yss** [comment]
Resolved @domenkozar

**Pull request merged**